lib/in/create-package-summary.js

```diff
@@ -30,6 +31,15 @@ function createPackageSummary(moduleName, currentState) {
         return false;
     }
 
+    // Ignore specified '--ignore' package globs
+    const ignore = currentState.get('ignore');
+    if (ignore) {
+        const ignoreMatch = Array.isArray(ignore) ? ignore.some(ignoredModule => minimatch(moduleName, ignoredModule)) : minimatch(moduleName, ignore);
```

Comment:
You can pass the ignore array to depcheck function. Then, there is no need to depends on minimatch in npm-check.